tests/test-backend-ops.cpp

```diff
@@ -1674,21 +1674,28 @@ struct test_silu_back : public test_case {
 struct test_norm : public test_case {
     const ggml_type type;
     const std::array<int64_t, 4> ne;
-    float eps;
+    const bool v; // whether a is a non-contiguous view
```

Comment:
Just making a note here, we cannot keep adding parameters like to every op test case, it makes them much more complex. This will need to be refactored at some point and replaced with a generic way to create non-contiguous views for the op parameters.